wazp/pages/02_ROI.py

```diff
@@ -35,12 +35,12 @@
 instructions = (
     "#### Instructions\n"
```

Comment:
I'd suggest we remove the instructions for now and ask Sanna for feedback. I think it's nice to keep it minimal and uncluttered to start with. I'd say with the new layout we may be able to do away with the instructions

Reply:
Will do. Was half of a mind to remove them myself anyway!